### ISSUE TITLE
How to use command `pymobiledevice3 developer dvt simulate-location set --rsd HOST PORT -- x y`

### ISSUE THREAD

**Issue by xxCabin** [doronz88/pymobiledevice3#509]
**Test environment**
- Host OS version.
- Target device model and iOS version.

**Describe the bug**

**To Reproduce**

**Expected behavior**

**Logs**
If applicable, add logs including backtraces for the given error that help explain your problem.

**Additional context**

**Comment by xxCabin** [comment]
`pymobiledevice3 mounter auto-mount` is success；and how do I get HOST PORT ?

**Comment by fcyctop** [comment]
> `pymobiledevice3 mounter auto-mount` is success；and how do I get HOST PORT ?

try this. https://github.com/doronz88/pymobiledevice3/issues/502

**Comment by xxCabin** [comment]
> > `pymobiledevice3 mounter auto-mount` is success；and how do I get HOST PORT ?
> 
> try this. #502

thanks.
and i have try it. now i can got this:

```
{

    "clientParameters": {

        "address": "fdb1:de43:283::2",

        "mtu": 1280,

        "netmask": "ffff:ffff:ffff:ffff::"

    },

    "serverAddress": "fdb1:de43:283::1",

    "serverRSDPort": 50935,

    "type": "serverHandshakeResponse"

}
```

in document, i need got 
```
Timestamp                       Thread     Type        Activity             PID    TTL
2023-07-19 08:22:51.916784+0300 0x3058     Info        0x0                  599    0    remotepairingd: (RemotePairing) [com.apple.dt.remotepairing:networktunnelmanager] tunnel-1: Tunnel established for interface: utun3, local fd41:8efc:c0f8::2 -> fd41:8efc:c0f8::1
2023-07-19 08:22:51.917310+0300 0x559c     Info        0x0                  599    0    remotepairingd: [com.apple.dt.remotepairing:remotepairingd] device-0: Tunnel established - interface: utun3, local fd41:8efc:c0f8::2-> remote fd41:8efc:c0f8::1
2023-07-19 08:22:51.917414+0300 0x559c     Info        0x0                  599    0    remotepairingd: [com.apple.dt.remotepairing:remotepairingd] device-0: Creating RSD backend client device for server port 60364
```

but now, i can not got it througt command, it`s return nothing
```
log stream --debug --info --predicate 'eventMessage LIKE "*Tunnel established*" OR eventMessage LIKE "*for server port*"'
```

what should i do next?

**Comment by xxCabin** [comment]
process remoted create utun5 failed in system log.

```
Aug 10 14:44:20 iPhone-3 configd[43] <Notice>: network changed
Aug 10 14:44:20 iPhone-3 remoted(libxpc.dylib)[51] <Notice>: [0xed080d2e0] activating connection: mach=false listener=false peer=true name=com.apple.remoted.coredevice.peer.0xed080d2e0
Aug 10 14:44:20 iPhone-3 remoted[51] <Notice>: add server listener start
Aug 10 14:44:20 iPhone-3 remoted[51] <Notice>: **Creating new listener device for interface: utun5**
Aug 10 14:44:20 iPhone-3 remoted[51] <Notice>: coredevice-15> Attached
Aug 10 14:44:20 iPhone-3 remoted[51] <Notice>: **Attached new listener device: coredevice-15 for interface: utun5**
Aug 10 14:44:20 iPhone-3 remoted[51] <Notice>: **add server listener done**
Aug 10 14:44:20 iPhone-3 remoted(Network)[51] <Notice>: [L49 tcp, definite, server, attribution: developer, context: Default Network Context (private), proc: 3A376F6F-9EFA-383A-BFF4-BB85C4A3E311, prohibit fallback, local address: ::.0, required interface: utun5(27)] start
Aug 10 14:44:20 iPhone-3 remoted[51] <Notice>: coredevice-15> **needsConnect**
Aug 10 14:44:20 iPhone-3 remoted[51] <Notice>: coredevice-15> **server waiting for connection from client**
Aug 10 14:44:20 iPhone-3 configd(Network)[43] <Notice>: network_config_check_mobile_asset Checking mobile asset
Aug 10 14:44:20 iPhone-3 configd(Network)[43] <Notice>: network_config_run_probes Running network probes
Aug 10 14:44:20 iPhone-3 remotepairingdeviced(RemotePairingDevice)[276] <Error>: VirtualInterface: Read handler called without completion handler, dropping 1 packets
Aug 10 14:44:20 iPhone-3 remoted(Network)[51] <Notice>: nw_listener_reconcile_inboxes_on_queue [L49] Started inbox socket: 16, endpoint: ::.50950@utun5, interface: utun5 with parameters tcp, interface: utun5, local: ::.0, definite, attribution: developer, server
Aug 10 14:44:20 iPhone-3 remoted(Network)[51] <Notice>: nw_listener_set_state_on_queue [L49] reporting state ready ((null))
Aug 10 14:44:20 iPhone-3 remotepairingdeviced(RemotePairingDevice)[276] <Notice>: A41916FF-ABA1-4FF3-B475-F9E6B347D670: Packet transfer stats for <private>/<private>: written-to-interface=0, written-to-connection=0, read-from-interface=0, read-from-connection=0
Aug 10 14:44:20 iPhone-3 remotepairingdeviced(RemotePairingDevice)[276] <Notice>: A41916FF-ABA1-4FF3-B475-F9E6B347D670: Byte transfer stats for <private>/<private>: written-to-interface=0, written-to-connection=0, read-from-interface=0, read-from-connection=0
Aug 10 14:44:20 iPhone-3 remotepairingdeviced(RemotePairingDevice)[276] <Error>: A41916FF-ABA1-4FF3-B475-F9E6B347D670: **Dropped 1 invalid packets received from virtual interface** <private>/<private>
Aug 10 14:44:20 iPhone-3 remotepairingdeviced(RemotePairingDevice)[276] <Error>: A41916FF-ABA1-4FF3-B475
-F9E6B347D670: **Dropped 1 invalid packets received over tunnel connection for interface** <private>/<private>
Aug 10 14:44:20 iPhone-3 remotepairingdeviced(libquic.dylib)[276] <Notice>: quic_frame_process_CONNECTION_CLOSE [C97:1] [2b556d9254403fb0-6c1d3b2e7582b3d2] **received APPLICATION_CLOSE code 0 reason** ""
Aug 10 14:44:20 iPhone-3 remotepairingdeviced(Network)[276] <Notice>: [C98 FE60469A-1992-4FA3-99C9-64735A7B4FC6 IPv6#02bde92c%anpi0.58623 quic, local: fe80::e4df:81ff:fef1:ffd2%anpi0.63153, tls, definite, no cellular, attribution: developer, server, prohibit joining] cancel
Aug 10 14:44:20 iPhone-3 SpringBoard(LocalAuthenticationPrivateUI)[593] <Notice>: CubicBSplineRenderer (0x718cb7c50): missing drawable, dropping frame 13035.
Aug 10 14:44:20 iPhone-3 wifid(CoreWiFi)[50] <Notice>: [corewifi] SCNetworkConfiguration event: keys=(
    "State:/Network/Global/IPv4"
)
Aug 10 14:44:20 iPhone-3 nehelper(Network)[113] <Notice>: -[NWPrivilegedHelper handleRequest:onConnection:] Handling check_mobile_asset from pid 43
Aug 10 14:44:20 iPhone-3 nehelper(Network)[113] <Notice>: -[NWPrivilegedHelper handleRequest:onConnection:] Returning 0 for check_mobile_asset to pid 43
Aug 10 14:44:20 iPhone-3 nehelper(Network)[113] <Notice>: -[NWPrivilegedHelper handleRequest:onConnection:] Handling run_probes from pid 43
Aug 10 14:44:20 iPhone-3 nehelper(Network)[113] <Notice>: handleRunProbes created transaction <private>
Aug 10 14:44:20 iPhone-3 nehelper(Network)[113] <Notice>: -[NWPrivilegedHelper handleRequest:onConnection:] Returning 0 for run_probes to pid 43
Aug 10 14:44:20 iPhone-3 nehelper(Network)[113] <Notice>: saveAndPostNetworkdSettings skipping OSLog settings update due to no logging changes
Aug 10 14:44:20 iPhone-3 backboardd(QuartzCore)[65] <Notice>: 111646468_DEBUG: can_update returning false due to is_enabled:0, mode.is_null:0, _frozen:0, flipbook_active:1, disables_updates:0
Aug 10 14:44:20 iPhone-3 backboardd(QuartzCore)[65] <Notice>: 111646468_DEBUG: render interval reason: 3
Aug 10 14:44:20 iPhone-3 remotepairingdeviced(Network)[276] <Notice>: [C98 FE60469A-1992-4FA3-99C9-64735A7B4FC6 IPv6#02bde92c%anpi0.58623 quic, local: fe80::e4df:81ff:fef1:ffd2%anpi0.63153, tls, definite, no cellular, attribution: developer, server, prohibit joining] cancelled
	[C98 E4C6E7F1-B4B7-449B-9C9C-722D1957F18E fe80::e4df:81ff:fef1:ffd2%anpi0.63153<->IPv6#02bde92c%anpi0.58623]
	Connected Path: satisfied (Path is satisfied), viable, interface: anpi0, scoped
	Duration: 0.016s, QUIC @0.000s took 0.000s, TLS 1.3 took 0.008s
	bytes in/out: 2565/3593, packets in/out: 7/10, rtt: 0.002s, retransmitted bytes: 0, out-of-order bytes: 1242
	ecn packets sent/acked/marked/lost: 0/0/0/0
Aug 10 14:44:20 iPhone-3 remotepairingdeviced(Network)[276] <Notice>: nw_flow_disconnected [C98 IPv6#02bde92c%anpi0.58623 cancelled socket-flow ((null))] Output protocol disconnected
Aug 10 14:44:20 iPhone-3 remotepairingdeviced(Network)[276] <Notice>: nw_connection_report_state_with_handler_on_nw_queue [C98] reporting state cancelled
Aug 10 14:44:20 iPhone-3 remotepairingdeviced(Network)[276] <Notice>: -[NWConcrete_nw_listener handleInboxFailed:error:] [L54] inbox failed: <protocol: quic>
Aug 10 14:44:20 iPhone-3 remotepairingdeviced(Network)[276] <Notice>: nw_listener_set_state_on_queue [L54] reporting state waiting ((null))
Aug 10 14:44:20 iPhone-3 remotepairingdeviced(libboringssl.dylib)[276] <Notice>: nw_protocol_boringssl_error(1902) [C98:1][0xc2ca38f20] Lower protocol stack error post TLS handshake. [57: <private>]
Aug 10 14:44:20 iPhone-3 remotepairingdeviced(Network)[276] <Notice>: nw_flow_disconnected [C100 IPv6#02bde92c%anpi0.58623 ready socket-flow (satisfied (Path is satisfied), viable, interface: anpi0, scoped)] Output protocol disconnected
Aug 10 14:44:20 iPhone-3 remotepairingdeviced(Network)[276] <Notice>: nw_flow_disconnected [C97 IPv6#02bde92c%anpi0.58623 ready socket-flow (satisfied (Path is satisfied), viable, interface: anpi0, scoped)] Output protocol disconnected
Aug 10 14:44:20 iPhone-3 remotepairingdeviced(Network)[276] <Notice>: 
nw_flow_disconnected [C99 IPv6#02bde92c%anpi0.58623 ready socket-flow (satisfied (Path is satisfied), viable, interface: anpi0, scoped)] Output protocol disconnected
Aug 10 14:44:20 iPhone-3 remotepairingdeviced(Network)[276] <Error>: nw_read_request_report [C99] Receive failed with error "Socket is not connected"
Aug 10 14:44:20 iPhone-3 remotepairingdeviced(libquic.dylib)[276] <Notice>: quic_conn_log_summary [C97:1] [2b556d9254403fb0-6c1d3b2e7582b3d2] 
	Connection attempts: 1, RETRY received: no, PTOs: 1
	Early data: no, Keep-alives sent/acknowledged: 0/0, ECN state: unsupported, L4S: disabled
	RTT: base 1 ms, network 4 ms, latest 4 ms, minimum 1 ms, smoothed 2 ms (variance 1 ms)
	Path MTU: 1500, minimum MSS: 1232
	Migration events: 0, paths validated: 0
	Inbound unidirectional/bidirectional streams: 0/1
	Outbound unidirectional/bidirectional streams: 0/1
	DATA_BLOCKED frames sent/received: 0/0
	STREAM_DATA_BLOCKED frames sent/received: 0/0
Aug 10 14:44:20 iPhone-3 SpringBoard(WiFiKit)[593] <Error>: CWFEventTypeIPChanged - event='<private>'
Aug 10 14:44:20 iPhone-3 remotepairingdeviced(RemotePairingDevice)[276] <Error>: A41916FF-ABA1-4FF3-B475-F9E6B347D670: failed with error: <private>
Aug 10 14:44:20 iPhone-3 remotepairingdeviced[276] <Notice>: A41916FF-ABA1-4FF3-B475-F9E6B347D670: Cancelling tunnel with error: Optional(Error Domain=NSPOSIXErrorDomain Code=57 "Socket is not connected")
Aug 10 14:44:20 iPhone-3 remotepairingdeviced(Network)[276] <Notice>: nw_connection_group_cancel_block_invoke [G16] cancel
Aug 10 14:44:20 iPhone-3 remotepairingdeviced(Network)[276] <Notice>: nw_listener_cancel_block_invoke [L53] cancel
Aug 10 14:44:20 iPhone-3 remotepairingdeviced(libxpc.dylib)[276] <Notice>: [0xc2cb25be0] invalidated on xpc_connection_cancel()
Aug 10 14:44:20 iPhone-3 remotepairingdeviced(Network)[276] <Notice>: nw_listener_resume_inbox [L53] Resumed for socket: 6, endpoint: ::.63153, interface: <NULL>
Aug 10 14:44:20 iPhone-3 remotepairingdeviced(Network)[276] <Notice>: nw_listener_cancel_all_inboxes_on_queue_block_invoke [L53] cancelling all inboxes: socket: 6, endpoint: ::.63153, interface: <NULL>
Aug 10 14:44:20 iPhone-3 mDNSResponder[137] <Notice>: *** Network Configuration Change *** SC key: State:/Network/Global/IPv4
Aug 10 14:44:20 iPhone-3 mDNSResponder[137] <Notice>: *** Network Configuration Change *** 1 change delay 25
Aug 10 14:44:20 iPhone-3 remotepairingdeviced[276] <Notice>: A41916FF-ABA1-4FF3-B475-F9E6B347D670: Cancelling tunnel with error: Optional(RemotePairingDevice.RemotePairingError(errorCode: 4, errorUserInfo: ["NSLocalizedDescription": "A networking error occurred."]))
Aug 10 14:44:20 iPhone-3 remotepairingdeviced(Network)[276] <Notice>: nw_listener_cancel_block_invoke [L54] cancel
Aug 10 14:44:20 iPhone-3 remotepairingdeviced(Network)[276] <Notice>: nw_listener_set_state_on_queue [L54] reporting state cancelled ((null))
Aug 10 14:44:20 iPhone-3 remotepairingdeviced[276] <Error>: A41916FF-ABA1-4FF3-B475-F9E6B347D670: Tunnel recieved error on remoted XPC connection: XPCError(errorCode: 1001, errorUserInfo: ["NSLocalizedDescription": "The connection was invalidated."])
Aug 10 14:44:20 iPhone-3 remotepairingdeviced(Network)[276] <Notice>: [C97 6D20320E-0A73-49CC-8D6E-B0622E54B651 IPv6#02bde92c%anpi0.58623 quic, local: fe80::e4df:81ff:fef1:ffd2%anpi0.63153, definite, no cellular, attribution: developer, server, prohibit joining] cancel
Aug 10 14:44:20 iPhone-3 remotepairingdeviced[276] <Notice>: A41916FF-ABA1-4FF3-B475-F9E6B347D670: Cancelling tunnel with error: Optional(Mercury.XPCError(errorCode: 1001, errorUserInfo: ["NSLocalizedDescription": "The connection was invalidated."]))
Aug 10 14:44:20 iPhone-3 remotepairingdeviced(Network)[276] <Notice>: [C97 6D20320E-0A73-49CC-8D6E-B0622E54B651 IPv6#02bde92c%anpi0.58623 quic, local: fe80::e4df:81ff:fef1:ffd2%anpi0.63153, definite, no cellular, attribution: developer, server, prohibit joining] cancelled
	[C97 E4C6E7F1-B4B7-449B-9C9C-722D1957F18E fe80::e4df:81ff:fef1:ffd2%anpi0.63153<->IPv6#02bde92c%anpi0.58623]
	Connected Path: sat
isfied (Path is satisfied), viable, interface: anpi0, scoped
	Privacy Stance: Not Eligible
	Duration: 0.020s, QUIC @0.000s took 0.009s
	bytes in/out: 2565/3593, packets in/out: 7/10, rtt: 0.002s, retransmitted bytes: 0, out-of-order bytes: 1242
	ecn packets sent/acked/marked/lost: 0/0/0/0
Aug 10 14:44:20 iPhone-3 remotepairingdeviced(NetworkExtension)[276] <Notice>: Tearing down utun5
Aug 10 14:44:20 iPhone-3 remotepairingdeviced(libquic.dylib)[276] <Notice>: quic_stream_send_reset_stream [C97:1] [2b556d9254403fb0-6c1d3b2e7582b3d2] [S1] sending RESET_STREAM
Aug 10 14:44:20 iPhone-3 kernel[0] <Error>: utun_start: ctl_getenqueuepacketcount returned error 22
Aug 10 14:44:20 iPhone-3 kernel[0] <Error>: utun_start: ctl_getenqueuepacketcount returned error 22
Aug 10 14:44:20 iPhone-3 identityservicesd[61] <Notice>: Client stopped flow to endpoint (null)
Aug 10 14:44:20 iPhone-3 identityservicesd(IDSFoundation)[61] <Notice>: Parameters string doesn't have enough fields (0) for the three required parameters
Aug 10 14:44:20 iPhone-3 identityservicesd[61] <Notice>: closeNexusWithOptions: Client closed nexus: 3F8A6074-46F6-402F-97ED-29CEB08B88F6 options: {
    ClientUUID = "3F8A6074-46F6-402F-97ED-29CEB08B88F6";
    FromUser = 1;
}
Aug 10 14:44:20 iPhone-3 identityservicesd[61] <Notice>: nw_path_copy_path_for_client failed for client 3F8A6074-46F6-402F-97ED-29CEB08B88F6
Aug 10 14:44:20 iPhone-3 identityservicesd[61] <Notice>: closeNexusWithOptions: cannot find mutiplexer for client 3F8A6074-46F6-402F-97ED-29CEB08B88F6 Registered multiplexers {
}
Aug 10 14:44:20 iPhone-3 SpringBoard(WiFiKit)[593] <Error>: CWFEventTypeIPChanged - event='<private>'
Aug 10 14:44:20 iPhone-3 wifid(CoreWiFi)[50] <Notice>: [corewifi] SCNetworkConfiguration event: keys=(
    "State:/Network/Interface"
)
Aug 10 14:44:20 iPhone-3 nehelper(Network)[113] <Notice>: nwphRunProbes Not selected to run ECH canary. Finished with transaction <private>
Aug 10 14:44:20 iPhone-3 mDNSResponder[137] <Notice>: SetNetworkChanged: Scheduling in 25 ticks
Aug 10 14:44:20 iPhone-3 mDNSResponder[137] <Notice>: [R1655] DNSServiceRegister(<mask.hash: 'zRhKQmJ4Q59PdTJkHZu7rw=='>(ea5ea4c7), 63153) STOP PID[276](remotepairingde)
Aug 10 14:44:20 iPhone-3 mDNSResponder[137] <Notice>: DecrementAutoTargetServices: AutoTargetServices 3 Record <mask.hash: 'orZJhK1XxV0Jn9XBvvB/qw=='>
Aug 10 14:44:20 iPhone-3 mDNSResponder[137] <Notice>: DecrementAutoTargetServices: NumAllInterfaceRecords 56 NumAllInterfaceQuestions 4 <mask.hash: 'orZJhK1XxV0Jn9XBvvB/qw=='>
Aug 10 14:44:20 iPhone-3 mDNSResponder[137] <Notice>: mDNS_Deregister_internal: callback with mStatus_MemFree for <mask.hash: 'orZJhK1XxV0Jn9XBvvB/qw=='>
Aug 10 14:44:20 iPhone-3 mDNSResponder[137] <Notice>: DecrementAutoTargetServices: NumAllInterfaceRecords 55 NumAllInterfaceQuestions 4 <mask.hash: '9NUDYmTMasTGHpHh44sNGw=='>
Aug 10 14:44:20 iPhone-3 mDNSResponder[137] <Notice>: mDNS_Deregister_internal: callback with mStatus_MemFree for <mask.hash: '9NUDYmTMasTGHpHh44sNGw=='>
Aug 10 14:44:20 iPhone-3 remoted(libxpc.dylib)[51] <Notice>: [0xed080d2e0] invalidated after getting a no-senders notification - client is gone
Aug 10 14:44:20 iPhone-3 remoted[51] <Notice>: Peer connection canceled, disconnecting device: coredevice-15
Aug 10 14:44:20 iPhone-3 remoted(Network)[51] <Notice>: nw_listener_cancel_block_invoke [L49] cancel
Aug 10 14:44:20 iPhone-3 remoted(Network)[51] <Notice>: nw_listener_resume_inbox [L49] Resumed for socket: 16, endpoint: ::.50950@utun5, interface: utun5
Aug 10 14:44:20 iPhone-3 remoted(Network)[51] <Notice>: nw_listener_cancel_all_inboxes_on_queue_block_invoke [L49] cancelling all inboxes: socket: 16, endpoint: ::.50950@utun5, interface: utun5
Aug 10 14:44:20 iPhone-3 remoted(Network)[51] <Notice>: nw_listener_set_state_on_queue [L49] reporting state cancelled ((null))
Aug 10 14:44:20 iPhone-3 remotepairingdeviced(Network)[276] <Notice>: nw_connection_report_state_with_handler_on_nw_queue [C97] reporting state cancelled
Aug 10 14:44:20 iPhone-3 remotepairingdeviced(Network)[276] <Notice>
: nw_connection_group_set_state_locked [G16] state changed: ready -> cancelled (error (null))
Aug 10 14:44:20 iPhone-3 remotepairingdeviced(NetworkExtension)[276] <Notice>: Tearing down utun5
Aug 10 14:44:20 iPhone-3 remotepairingdeviced(NetworkExtension)[276] <Error>: SIOCGIFMTU failed: Device not configured
Aug 10 14:44:20 iPhone-3 remotepairingdeviced(NetworkExtension)[276] <Error>: NEVirtualInterfaceAdjustReadBufferSize: interface_get_mtu failed (6), defaulting to max mtu
Aug 10 14:44:20 iPhone-3 remotepairingdeviced(Network)[276] <Notice>: [C100 IPv6#02bde92c%anpi0.58623 failed socket-flow (satisfied (Path is satisfied), viable, interface: anpi0, scoped)] event: flow:disconnect @0.013s, error Socket is not connected
Aug 10 14:44:20 iPhone-3 remotepairingdeviced(Network)[276] <Notice>: nw_connection_report_state_with_handler_on_nw_queue [C100] **reporting state failed error Socket is not connected**
Aug 10 14:44:20 iPhone-3 configd[43] <Notice>: ioctl() failed: Device not configured
Aug 10 14:44:20 iPhone-3 configd[43] <Notice>: ioctl() failed: Device not configured
Aug 10 14:44:20 iPhone-3 remotepairingdeviced(Network)[276] <Notice>: [C99 IPv6#02bde92c%anpi0.58623 failed socket-flow (satisfied (Path is satisfied), viable, interface: anpi0, scoped)] event: flow:disconnect @0.014s, error Socket is not connected
Aug 10 14:44:20 iPhone-3 remotepairingdeviced(Network)[276] <Notice>: nw_connection_report_state_with_handler_on_nw_queue [C99] reporting state failed error Socket is not connected
Aug 10 14:44:20 iPhone-3 SpringBoard(LocalAuthenticationPrivateUI)[593] <Notice>: CubicBSplineRenderer (0x718cb7c50): missing drawable, dropping frame 13036.
Aug 10 14:44:20 iPhone-3 backboardd(QuartzCore)[65] <Notice>: 111646468_DEBUG: can_update returning false due to is_enabled:0, mode.is_null:0, _frozen:0, flipbook_active:1, disables_updates:0
Aug 10 14:44:20 iPhone-3 backboardd(QuartzCore)[65] <Notice>: 111646468_DEBUG: render interval reason: 3
Aug 10 14:44:20 iPhone-3 rapportd(CoreUtils)[90] <Notice>: LISTEN: Stop listen request from agent_client=<NWConcrete_nw_agent_client: 0xa92a33be0>
Aug 10 14:44:20 iPhone-3 rapportd(CoreUtils)[90] <Notice>: LISTEN: Not an application service advertise descriptor, found type=1
Aug 10 14:44:20 iPhone-3 rapportd(CoreUtils)[90] <Notice>: FLOW: Stop flow request from agent_client=<NWConcrete_nw_agent_client: 0xa92c5cc40> to endpoint=NULL
Aug 10 14:44:20 iPhone-3 rapportd(CoreUtils)[90] <Notice>: Looking up agent client from client=0xa92c5cc40
Aug 10 14:44:20 iPhone-3 rapportd(CoreUtils)[90] <Notice>: Found matching client RPNWAgentClient[0xa92c2c770] (OTCNX): < PID=276 flow_client=0xa92c5cc40 port=63153 >
Aug 10 14:44:20 iPhone-3 rapportd(CoreUtils)[90] <Notice>: Removing client RPNWAgentClient[0xa92c2c770] (OTCNX): < PID=276 port=63153 >
Aug 10 14:44:20 iPhone-3 mDNSResponder[137] <Notice>: DecrementAutoTargetServices: NumAllInterfaceRecords 54 NumAllInterfaceQuestions 4 <mask.hash: 'RNbG2GcoeXCNOGlM3WqElg=='>
Aug 10 14:44:20 iPhone-3 mDNSResponder[137] <Notice>: mDNS_Deregister_internal: callback with mStatus_MemFree for <mask.hash: 'RNbG2GcoeXCNOGlM3WqElg=='>
Aug 10 14:44:20 iPhone-3 mDNSResponder[137] <Notice>: DecrementAutoTargetServices: NumAllInterfaceRecords 53 NumAllInterfaceQuestions 4 <mask.hash: 'o5br45FQLtAFquCpJEXbjQ=='>
Aug 10 14:44:20 iPhone-3 mDNSResponder[137] <Notice>: mDNS_Deregister_internal: callback with mStatus_MemFree for <mask.hash: 'o5br45FQLtAFquCpJEXbjQ=='>
Aug 10 14:44:20 iPhone-3 mDNSResponder[137] <Notice>: ServiceCallback: All records unregistered for <mask.hash: 'giSGq3BmMtDfsG9XVRmBDg=='>
Aug 10 14:44:20 iPhone-3 mDNSResponder[137] <Notice>: [R0] DNSServiceRegister(<mask.hash: 'zRhKQmJ4Q59PdTJkHZu7rw=='>(ea5ea4c7), 63153) DEREGISTERED
Aug 10 14:44:20 iPhone-3 configd[43] <Notice>: siocsifnetsignature(utun5, IPv6, 0) failed: Device not configured
Aug 10 14:44:20 iPhone-3 configd[43] <Notice>: **utun5: ioctl(SIOCGIFTYPE) failed: Device not configured**

```
